x-pack/plugins/observability/public/components/threshold/hooks/use_metrics_explorer_chart_data.ts

```diff
@@ -43,8 +43,7 @@ export const useMetricsExplorerChartData = (
           ? {
               aggregation: 'custom',
               custom_metrics:
```

Comment:
should we rename `custom_metrics` here to `metrics`?

Reply:
No, this is actually the field that we pass to Infra API, and we need to keep it as it is since this API is also used in metric threshold. I created a ticket to move that API to observability, we can consider changing it then.

![image](https://github.com/elastic/kibana/assets/12370520/9844a05d-55c7-4855-9769-090d1ee24413)